Makefile

```diff
@@ -43,23 +43,32 @@ vet: ## Apply go vet to all go files
 build-e2e:
 	go test -c -o bin/e2e github.com/openshift/cluster-api-actuator-pkg/pkg/e2e
 
-.PHONY: test-e2e
-test-e2e: ## Run openshift specific e2e test
+define test =
 	go test -timeout 90m \
 		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
 		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
 		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
-		-ginkgo.v \
-		-args -v 5 -logtostderr true
+		-args -v 5 -logtostderr \
+		$1 $2 $3 $4 $5 $6 $7 $8 $9
+endef
+
+.PHONY: test-e2e
+test-e2e: ## Run openshift specific e2e test
+	# Run operator tests first to preserve logs for troubleshooting test
+	# failures and flakes.
+	# Feature:Operator tests remove deployments. Thus loosing all the logs
+	# previously acquired.
+	$(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.focus "Feature:Operators")
+	$(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.skip "Feature:Operators")
 
 .PHONY: k8s-e2e
 k8s-e2e: ## Run k8s specific e2e test
-	go test -timeout 30m \
-		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
-		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
-		-machine-api-namespace $${NAMESPACE:-kube-system} \
-		-ginkgo.v \
-		-args -v 5 -logtostderr true
+	# Run operator tests first to preserve logs for troubleshooting test
+	# failures and flakes.
+	# Feature:Operator tests remove deployments. Thus loosing all the logs
+	# previously acquired.
+	$(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.focus "Feature:Operators")
```

Comment:
`NAMESPACE` needs to be set to `kube-system` for both invocations:
```
NAMESPACE=kube-system $(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.focus "Feature:Operators")
NAMESPACE=kube-system $(call test,-ginkgo.v,-ginkgo.noColor=true,-ginkgo.skip "Feature:Operators")
```

Reply:
Done.